website/content/api-docs/index.mdx

```diff
@@ -75,6 +75,29 @@ $ curl \
     http://127.0.0.1:8500/v1/kv/foo
 ```
 
+## Accepts URL-Encoded Resource Names
+
+Some Consul HTTP API endpoints require resource names in the URL. To pass a resource
+name containing characters that are invalid in a URL path segment, such as `/` or ` `,
+the resource name must be URL-encoded into the HTTP API call URL.
+
+Though any resource name *can* be used with the HTTP API if URL-encoded,
+we typically recommend using resource names that don't require URL-encoding anyways.
```

Comment:
I think the first paragraph isn't really necessary - it was a bug that Consul did not decode correctly but _generally_ people would expect most servers to respect URL-encoded paths

```suggestion
Consul HTTP API endpoints can parse URL-encoded resource names but we typically 
recommend using names that will not require URL-encoding.
```

Reply:
Though being concise is important, I don't want to be concise at the expense of assuming what users will know or expect. If our assumptions are wrong for just 25% of users, that's still a lot of users who may be confused.